package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "icaredata-client",
+  "version": "0.0.0",
+  "description": "A command-line interface that posts FHIR Messages from a file system to the ICAREdata API",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ICAREdata/icaredata-client"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "axios": "^0.19.0",
+    "commander": "^4.0.1",
+    "fs-extra": "^8.1.0"
```

Comment:
Do you actually need this or is fs sufficient? I don't see anything being used here that fs doesn't have

Reply:
You know, I think you're right. I imported it out of habit, without really needing it, it seems. Let me get rid of it now.